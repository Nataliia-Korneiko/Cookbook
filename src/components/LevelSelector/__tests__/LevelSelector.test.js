/* eslint-disable react/jsx-props-no-spreading */
import React from 'react';
import { shallow } from 'enzyme';
import LevelSelector from '../LevelSelector';

describe('LevelSelector', () => {
  const props = {
    options: ['low', 'normal', 'high'],
    value: '',
    onChange: () => null,
  };

  const newLevelSelector = shallow(<LevelSelector {...props} />);

  it('renders a properly', () => {
    expect(newLevelSelector).toMatchSnapshot();
  });

  it('renders one select', () => {
    expect(newLevelSelector.find('select[name="level"]')).toHaveLength(1);
  });

  it('renders three options', () => {
    expect(newLevelSelector.find('option')).toHaveLength(3);
  });
});
