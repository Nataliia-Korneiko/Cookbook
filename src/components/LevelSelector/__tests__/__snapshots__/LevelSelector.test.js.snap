// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LevelSelector renders a properly 1`] = `ShallowWrapper {}`;
